e2e/README.md

```diff
@@ -0,0 +1,41 @@
+# E2E tests
+
+VMClarity's end-to-end tests use the [Ginkgo](https://onsi.github.io/ginkgo/) test framework and the [Gomega](https://onsi.github.io/gomega/) matcher/assertion library to leverage behavior driven testing (BDD).
+
+This test module is composed by:
+
+* a `testenv.go` file where the test environment is configured,
+* a `docker-compose.override.yml` file which will merge and override the installation compose file to run end-to-end tests,
+* a `suite_test.go` file where the steps to perform the setup and cleanup of the Ginkgo test suite are defined,
+* a `helper.go` file where common methods are defined,
+* and test case files.
+
+## Run test
```

Comment:
Can we please add sections to the contributing guide explaining the new make target, and also linking to this document for more details. This is what we have in KubeClarity:
https://github.com/openclarity/kubeclarity/blob/main/CONTRIBUTING.md#testing-end-to-end

Reply:
great suggestion! 👍